django/urls/__init__.py

```diff
@@ -1,12 +1,13 @@
 from .base import (
     clear_script_prefix, clear_url_caches, get_script_prefix, get_urlconf,
-    is_valid_path, resolve, reverse, reverse_lazy, set_script_prefix,
-    set_urlconf, translate_url,
+    is_valid_path, path, re_path, resolve, reverse, reverse_lazy,
+    set_script_prefix, set_urlconf, translate_url,
 )
+from .converters import register_converter
 from .exceptions import NoReverseMatch, Resolver404
 from .resolvers import (
-    LocaleRegexProvider, LocaleRegexURLResolver, RegexURLPattern,
```

Comment:
I did some [quick google searches](site:github.com LocaleRegexProvider) to see if this was used, and I just one: https://github.com/zulip/zulip/blob/master/zerver/lib/integrations.py